core/snippets/snippets/import.snippet

```diff
@@ -12,3 +12,8 @@ language: python
 -
 import $0
 ---
+
+language: r
+-
+library(${1:tidyverse}${2:, warn.conflicts = FALSE})
```

Comment:
Why `tidyverse` as the placeholder?

Reply:
One of, if not the, most commonly imported packages in R. Considered almost a second standard library. Seeing it there as a placeholder will indicate that you should put your package here, because it is such a recognisable package name.

Reply:
We talked about this in the community to meetup and we don't think that this default should be used.